packages/rule-stylesheet-limits/src/rule.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * @fileoverview Checks if CSS exceeds known stylesheet limits
+ */
+
+import { Category } from 'sonarwhal/dist/src/lib/enums/category';
+import { RuleContext } from 'sonarwhal/dist/src/lib/rule-context';
+// The list of types depends on the events you want to capture.
+import { IRule, ScanEnd, RuleMetadata } from 'sonarwhal/dist/src/lib/types';
+import { debug as d } from 'sonarwhal/dist/src/lib/utils/debug';
+import { RuleScope } from 'sonarwhal/dist/src/lib/enums/rulescope';
+
+const debug: debug.IDebugger = d(__filename);
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class StylesheetLimitsRule implements IRule {
+
+    public static readonly meta: RuleMetadata = {
+        docs: {
+            category: Category.interoperability,
+            description: `Checks if CSS exceeds known stylesheet limits`
+        },
+        id: 'stylesheet-limits',
+        schema: [{
+            additionalProperties: false,
+            properties: {
+                maxRules: { type: 'number' },
+                maxSheets: { type: 'number' },
+                maxImports: { type: 'number' }
+            },
+            type: ['object', 'null']
+        }],
+        scope: RuleScope.any
```

Comment:
Looking at how you get the data I believe the scope should be `browser`.

Reply:
I see `any`, `local`, and `site` and options for `RuleScope`. Were you referring to `site`?

I should also note that aside from the `@import` limitation mentioned in the PR description, `jsdom` does parse and populate the `document.styleSheets` collection meaning much of the testing can still be done in the `local` scope if I'm understanding the environment correctly.

Reply:
You are correct, it's `site`.

`local` reads files individually from the disk so it will not work for now unless we add a css parser.